icaruscode/IcarusObj/PMTBeamSignal.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file   icaruscode/IcarusObj/PMTBeamSignal.h
+ * @brief  Holds the event-by-event RWM or EW times
+ * @author Matteo Vicenzi (mvicenzi@bnl.gov)
+ * @date   March 14 2024
+ */
+
+#ifndef ICARUSCODE_ICARUSOBJ_PMTBEAMSIGNAL_H
+#define ICARUSCODE_ICARUSOBJ_PMTBEAMSIGNAL_H
+
+// C/C++ standard libraries
+#include <limits>
+
+namespace icarus::timing{
+		
+        /// Special value to denote no special channel information
+        static constexpr auto NoChannel = std::numeric_limits<unsigned int>::max();
+        /// Special value to denote no time channel information
+        static constexpr double NoTime = 0.;
+        /// Special value to denote no sample information
+        static constexpr size_t NoSample = 0;
+
+	struct PMTBeamSignal {
+		
+		/// Special channel this time was extracted from
+		unsigned int specialChannel = NoChannel;
+	 	
+                /// Board on which the special channel is on
```

Comment:
Can you add in the comment an example of the expected format? e.g. `EW-TOP-A`, or `ewtop00`, etc. Also, if you recommend this to be the same as the value reported by the channel mapping database ([`icarusDB::PMTChannelInfo_t::digitizerLabel`](https://github.com/SBNSoftware/icaruscode/blob/8f2c668ce60e089a06ef3ed6b30a7686df738e1a/icaruscode/Decode/ChannelMapping/ICARUSChannelMapDataTypes.h#L80)),  say that explicitly here or in the class description.

Reply:
Done!